ui/src/alerting/utils/history.ts

```diff
@@ -65,7 +64,7 @@ export const loadNotifications = (
   const fluxFilter = filter ? searchExprToFlux(renameTagKeys(filter)) : null
 
   const query = `
-from(bucket: "${NOTIFICATION_BUCKET}")
+from(bucket: "${MONITORING_BUCKET}")
   |> range(start: ${start}, stop: ${Math.round(until / 1000)})
   |> filter(fn: (r) => r._field == "_message")
```

Comment:
Looks like this was messed up before, but in any case I think it now needs to be:
```suggestion
  |> filter(fn: (r) => r._measurement == "notifications")
```